h5grove/encoders.py

```diff
@@ -23,10 +24,11 @@ def orjson_default(o: Any) -> Union[list, float, str, None]:
     if isinstance(o, np.number) and o.dtype.kind == "f" and o.itemsize > 8:
         # Force conversion of float >64bits to native float even if it means losing precision
         return float(o)
+    if isinstance(o, numbers.Complex):
+        # Force conversion of float >64bits to native float even if it means losing precision
+        return [float(o.real), float(o.imag)]
```

Comment:
Opening as draft as this casting to float64 doesn't seem to work with complex256 values. When I open `/complex256_scalar`, I still get `[0.0,null]` instead of `[0.0,inf]` (like with `/float128_2D`).